assets/js/googlesitekit/store/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import SiteKitRegistry from 'googlesitekit-data';
+import siteStore, { STORE_NAME } from 'assets/js/googlesitekit/site';
+
+const createStore = ( registry = SiteKitRegistry ) => {
+	registry.registerStore( STORE_NAME, siteStore );
+};
+
+export default createStore;
```

Comment:
Why is this file here? Doesn't seem to be used anywhere and wasn't discussed before I believe. There should just be a `googlesitekit-site.js` file that calls `SiteKitRegistry.registerStore( STORE_NAME, siteStore )`. No need for a separate `store/index.js` module, and particularly, it shouldn't use that name as there will be multiple stores.

Reply:
Yeah, typically the main `store` file in a Redux setup in the `createStore` function that loads all stores, but we're going for a different approach here and this was me feeling some things out, but you're right—this can go 😄